src/Uno.UI/Media/NativeRenderTransformAdapter.iOSmacOS.cs

```diff
@@ -64,6 +67,17 @@ partial class NativeRenderTransformAdapter
 			}
 		}
 
+		private void InitializeOrigin()
+		{
+			var oldFrame = Owner.Layer.Frame;
+
+			Owner.Layer.AnchorPoint = CurrentOrigin;
+
+			// Restore the old frame to correct the offset potentially introduced by changing AnchorPoint. This is safe to do since we know
+			// that the transform is currently identity.
+			Owner.Layer.Frame = oldFrame;
```

Comment:
The `Transform` may be something else than identity.

Reply:
It should normally be identity unless I'm misreading something. We've just removed the previous transform if any, and haven't yet applied the new transform.

Reply:
Depends if we speak about the `Layer.Transform` or the `Transform` property of the object. I agree that the `Layer.Transform` should be identity (except if someone altered it using the native API instead of the Windows/UNO APIs, but it's probably not a concern). Conversely the `Transform` property is usually not identity at this point.

In fact I think that your comment is about the `Layer.Transform`, so it's  right, but we should probably explicit this.

Reply:
Gotcha. Yes, it's the `Layer.Transform' property we're concerned about, since setting `Frame` when `Layer.Transform` is non-identity gives undefined behaviour according to Apple's docs.